dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -25,8 +25,8 @@ disableJspRuntimeCompilation=Disable compilation of JSPs at runtime.
 extendedDocumentRoot.name=Extended document root
 extendedDocumentRoot=Directory that the JSP engine will search for additional JSP files to serve.
 
-jdkSourceLevel.name=JDK source level
-jdkSourceLevel=Default Java source level for JSPs compiled by the JSP engine.
+javaSourceLevel.name=Java source level
+javaSourceLevel=Default Java source level for JSPs compiled by the JSP engine.  Supports LTS releases: 8, 11, and 17.
```

Comment:
Do we have an issue opened to update for Java 21 once available?

Reply:
Also the copyright header date range should be updated.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/26088